bftengine/src/bftengine/messages/PrePrepareMsg.cpp

```diff
@@ -96,8 +98,10 @@ void PrePrepareMsg::validate(const ReplicasInfo& repInfo) const {
   const bool isNull = ((flags & 0x1) == 0);
   const bool isReady = (((flags >> 1) & 0x1) == 1);
   const uint16_t firstPath_ = ((flags >> 2) & 0x3);
-  const uint16_t reservedBits = (flags >> 4);
-
+  const bool isConsensusPP = (((flags >> 4) & 0x01) == 0x01);
+  const bool isDataPP = (((flags >> 4) & 0x10) == 0x10);
```

Comment:
Notice that if you attach the data to the consensusPP you can also have a data in the consensusPP (Foe example, as a result of the RFMI)